db/migrate/20150509223803_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.references :image, index: true, foreign_key: true
+      t.references :user, index: true, foreign_key: true
+      t.text :text
+
```

Comment:
Would it make sense to also store a score, in case we want users to be able to upvote/downvote comments later?